server/eb/README.md

```diff
@@ -120,6 +120,30 @@ ebextensions:
 Any additional config files intended for the `.ebextensions` directory of the artifact can be added
 to the `customize/ebextensions` directory.  Any file found here will be copied over.
 
+requirements.txt:
+
+A custom requirements.txt can be supplied in customize/requirements.txt.   
+This file fully specify the versions of all the python modules used by there server in the deployment.
```

Comment:
wording:   "This file must fully ..."
wording:  "... used by the server ..."

Reply:
done